docs/reference/callbacks.txt

```diff
@@ -10,6 +10,10 @@ Callbacks
    :depth: 2
    :class: singlecol
 
+Mongoid implements many of the `ActiveRecord callbacks
+<https://guides.rubyonrails.org/active_record_callbacks.html>`_.
```

Comment:
suggest to add `... but does not guarantee 100% feature parity.`

Reply:
This simply creates confusion. Any divergences in behavior that are significant enough for users to know should be documented.